<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TT_DialogProjectSettings</class>
 <widget class="QDialog" name="TT_DialogProjectSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="2,1">
     <item row="0" column="0">
      <widget class="QLabel" name="lb_scale">
       <property name="text">
        <string>Project scale</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cb_scale">
       <item>
        <property name="text">
         <string>1:100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1:200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1:500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lb_customScale">
       <property name="text">
        <string>Custom project scale</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_customScale">
       <property name="placeholderText">
        <string>1:125</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TT_DialogProjectSettings</receiver>
   <slot>slot_validateInputs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TT_DialogProjectSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_scale</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>TT_DialogProjectSettings</receiver>
   <slot>slot_cbScaleIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_customScale</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TT_DialogProjectSettings</receiver>
   <slot>slot_leCustomScaleTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_cbScaleIndexChanged(int)</slot>
  <slot>slot_leCustomScaleTextChanged(QString)</slot>
  <slot>slot_validateInputs()</slot>
 </slots>
</ui>
